packages/paste-website/src/components/assistant/AssistantCanvas.tsx

```diff
@@ -62,29 +60,51 @@ export const AssistantCanvas: React.FC<AssistantCanvasProps> = ({ selectedThread
     <Box ref={scrollerRef} tabIndex={0} overflowY="auto">
       <Box maxWidth="1000px" marginX="auto">
         {activeRun != null && <AssistantMessagePoller />}
-        <ChatLog ref={loggerRef}>
-          <ChatBookend>
-            <ChatBookendItem>
+        <AIChatLog ref={loggerRef}>
+          <Box display="flex" flexDirection="column" rowGap="space40">
+            <Text
+              as="span"
+              color="colorTextWeak"
+              fontSize="fontSize20"
+              lineHeight="lineHeight20"
+              fontWeight="fontWeightMedium"
+              textAlign="center"
+            >
               Welcome to the Paste Design System Assistant! We&apos;re excited to have you here.
-            </ChatBookendItem>
-          </ChatBookend>
-          <ChatBookend>
-            <ChatBookendItem>
+            </Text>
+            <Text
+              as="span"
+              color="colorTextWeak"
+              fontSize="fontSize20"
+              lineHeight="lineHeight20"
+              fontWeight="fontWeightMedium"
+              textAlign="center"
+            >
               Keep in mind that this is an experimental tool and so the information provided{" "}
-              <strong>may not be entirely accurate</strong>.
-            </ChatBookendItem>
-            <ChatBookendItem>
+              <Text as="span" fontWeight="fontWeightBold">
+                may not be entirely accurate
+              </Text>
```

Comment:
issue: Can you set the size and color of this to match the text around it?

<img width="719" alt="Screenshot 2024-10-31 at 12 38 48 PM" src="https://github.com/user-attachments/assets/421e0dc7-c7f8-44c2-b9c5-3d5a24321442">